index.html

```diff
@@ -276,5 +276,25 @@ <h3 class="drp_text">Skills</h3>
     </div>
 
   </section>
+  <section class="contactMe">
+      <h1 class="contactTxt">Contact me</h1>
+      <p 
+        class="Contact_body_text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+      </p>
+      <form class="	form" action="https://formspree.io/f/xnqrezql" method="post" >
+        <input type="text" id="name" name="user_name" placeholder="FullName"  required maxlength="30"/>
+        <input type="email" id="mail" name="user_email" placeholder="Email Address"  pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+.[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-])*$/" required />
+        <div class="conter">
+          <textarea id="textarea" cols="30" rows="10" placeholder="write something here" name="user_message" required maxlength="500"></textarea>
+        </div>
+        <div id="actin">
+          <button type="submit" class="normalButtnAbout" onclick=""> Get in Touch</button>
+        </div>
```

Comment:
Nice work adding `placeholders` to your input fields as requested by the previous reviewer😉 But just as the previous reviewer suggested, let's make our project look more like the design😉 Kindly take note of the images provided by the previous reviewer and make comparison😉 Take note of the `height` and `width` of each input fields and the position of the button😉😉